examples/bpk-component-datepicker/examples.js

```diff
@@ -429,128 +429,116 @@ class MultipleInputDatepicker extends Component {
 }
 
 const DefaultExample = () => (
-  <div id="application-element">
```

Comment:
This is actually unrelated to this PR, but thought I'd fix these examples quickly as due to the fact that the id is `application-element`, we cannot render datepicker on ios due to the logic [here](https://github.com/Skyscanner/backpack/blob/main/packages/bpk-scrim-utils/src/withScrim.js#L73) which adds `display: none` to the application element.